cime_config/acme/machines/Makefile

```diff
@@ -185,14 +185,40 @@ ifndef AR
    AR := ar
 endif
 
-ifdef NETCDF_PATH
-  ifndef INC_NETCDF
-    INC_NETCDF:=$(NETCDF_PATH)/include
-  endif
-  ifndef LIB_NETCDF
-    LIB_NETCDF:=$(NETCDF_PATH)/lib
+# To prevent breaking cleaning and building components which don't require NetCDF directly
+ifeq ($(if $(or $(filter pio, $(subst /,  , $(MAKECMDGOALS))), $(filter exec_se, $(subst /,  , $(MAKECMDGOALS)))), true, false), true)
```

Comment:
The variable MODEL in the makefile is what you want.   When pio is built MODEL=PIO1 or PIO2, when the executable is built MODEL = CIME_MODEL (acme or cesm)